packages/block-editor/src/components/inserter/block-patterns.js

```diff
@@ -64,20 +73,28 @@ function BlockPatterns( { patterns, onInsert } ) {
 		);
 	}, [] );
 
-	return (
-		<InserterPanel title={ _x( 'All', 'patterns' ) }>
-			{ patterns.map( ( pattern, index ) =>
+	return !! filteredPatterns.length ? (
+		<InserterPanel
+			title={
+				filterValue
+					? _x( 'Search Results', 'patterns' )
+					: _x( 'All', 'patterns' )
```

Comment:
I don't know that these strings need to be contextualized. Well, at least the first one doesn't seem like one that risks being ambiguous. "All" might be, but "patterns" doesn't really do much to help with the context. It might be something more like `_x( 'All', 'empty search query' )`. This would also allow the string to be reused, rather than having a bunch of unique strings for `_x( 'All', 'patterns' )`, `_x( 'All', 'blocks' )`, etc.

Reply:
This term  is only useful for patterns, it means "All categories" but it will probably be removed once we  introduce the support for Patterns categories.